src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/Logging/ActivityLogLoggerProvider.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.ComponentModel.Composition;
+using Microsoft.CodeAnalysis.Razor.Logging;
+using Microsoft.VisualStudio.Razor.Logging;
+
+namespace Microsoft.VisualStudio.LanguageServices.Razor.Logging;
+
+/// <summary>
+/// An <see cref="ILoggerProvider"/> that logs any warnings or errors to the Visual Studio Activity Log.
+/// </summary>
+[Export(typeof(ILoggerProvider))]
+[method: ImportingConstructor]
+internal sealed partial class ActivityLogLoggerProvider(RazorActivityLog activityLog) : ILoggerProvider
```

Comment:
Since I'm on high alert for this, is this going to cause any image loads in scenarios where we weren't doing that before?